web/app/Http/Controllers/AccountController.php

```diff
@@ -112,4 +115,76 @@ public function sendVerificationEmail(): Response
 
         return new Response('', 202);
     }
+
+    /**
+     * Starts password recovery.
+     * Endpoint: `POST:/account/start-recovery` | `accountStartRecovery`
+     */
+    public function startRecovery(Request $request): Response
+    {
+        // check if user is logged in
+        if ($this->guard->check()) {
+            return new Response('', 409);
+        }
+
+        $email = $request->input('email');
+
+        // no one is using this email, return 403
+        if (!UserValidation::isEmailTaken($email)) {
+            return new Response('', 403);
+        }
+
+        // send recovery email
+
+        $status = Password::sendResetLink($request->only('email'));
+
+        switch ($status) {
+            case Password::RESET_LINK_SENT:
+                return new Response('', 202);
+
+            case Password::INVALID_USER:
+                return new Response('', 403);
+
+            default:
+                return new Response('', 500);
+        }
+    }
+
+    /**
+     * Handles updating a password during password recovery.
+     * Not part of the "proper" API.
```

Comment:
What do you mean by this?

Reply:
It's not accessed via `/api--v0` or whatever. I suppose I could make it part of the actual API, but a token has to be provided and it's kind of unlike all the other API endpoints.